### PR TITLE
docs: add Seven-innovation-base to showcase

### PR DESCRIPTION
**Pull request by redhat123456**

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -2087,6 +2087,14 @@ const Users: User[] = [
     source: 'https://github.com/HiChen404/MyWikiSite',
     tags: ['opensource', 'personal'],
   },
+  {
+    title: '数学与计算科学学院创新实践基地',
+    description: '关于七院创新基地的网站',
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:2092]: Please provide the title/description in English.

File: website/src/data/users.tsx [hunk 2]
```diff
@@ -2087,6 +2087,14 @@ const Users: User[] = [
...
+    preview: require('./showcase/1.png'),
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:2093]: Please use a better file name (should be more obvious after you have an English title)

File: website/src/data/users.tsx [hunk 3]
```diff
@@ -2087,6 +2087,14 @@ const Users: User[] = [
...
+    website: 'https://七院创新基地.cn',
+    source: 'https://github.com/seven-innovation-base/official-website',
+    tags: ['opensource', 'study','university'],
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:2096]: Please only use known tags. All tags are listed at the start of the file.

File: website/src/data/users.tsx (hunk 1 above)
Comment (redhat123456) [reply website/src/data/users.tsx:2092]: 收到

File: website/src/data/users.tsx (hunk 2 above)
Comment (redhat123456) [reply website/src/data/users.tsx:2093]: 收到

File: website/src/data/users.tsx (hunk 3 above)
Comment (redhat123456) [reply website/src/data/users.tsx:2096]: 收到

**Comment by redhat123456** [comment]
改好了，请大佬过目  @Josh-Cena

**Comment by redhat123456** [comment]
这机器人看起来还挺有趣的，能否介绍一下

**Comment by Josh-Cena** [comment]
Please remember to sign the CLA, or I can't merge :) All the other stuff I can take care of before merging

**Comment by Josh-Cena** [comment]
> 这机器人看起来还挺有趣的，能否介绍一下

The CLA bot? It's just connected to Facebook's CLA registry, and every time you send a PR, it checks if the registry has a record associated with your email.

**Comment by redhat123456** [comment]
专业的回答

我也是搞数学的数学人，以后可以多多交流😁

**Comment by redhat123456** [comment]
谢谢大佬，我就说刚刚怎么就报错了

**Comment by Josh-Cena** [comment]
Yeah, now it's just the CLA left. If you've signed it it should be updated within a few hours. We can just wait :)

**Comment by redhat123456** [comment]
不知道怎么CLA签名 （ 😭

**Review by redhat123456 — COMMENTED** [review]
好

**Comment by Josh-Cena** [comment]
Select "individual" here: https://code.facebook.com/cla, provide your information, read the agreement, check the "I agree", and submit.

**Comment by redhat123456** [comment]
okok

**Comment by redhat123456** [comment]
明白了

**Review by Josh-Cena — APPROVED** [review]
Great, thanks!

File: admin/scripts/image-resize.mjs [hunk 4]
```diff
@@ -25,7 +25,7 @@ await Promise.all(
       new URL(`../../website/src/data/showcase/${img}`, import.meta.url),
     );
     const {width, height} = imageSize(imgPath);
-    if (width === 640 && height === 320) {
+    if (width === 640 && height === 320 && imgPath.endsWith('.png')) {
```
Comment (slorber) [inline admin/scripts/image-resize.mjs:28]: what's the purpose of this new condition?

Comment (Josh-Cena) [reply admin/scripts/image-resize.mjs:28]: We normalize JPEG images to PNG, but if an image has 640×320 dimensions it will be ignored and stay as JPEG.

**Pull request merged**